### ISSUE TITLE
Create standard CTMR R environment

### ISSUE THREAD

**Issue by boulund** [ctmrbio/dockerfiles#1]
Suggested packages (alphabetical order):

* [beeswarm](https://cran.r-project.org/web/packages/beeswarm/index.html)
* [DADA2](https://benjjneb.github.io/dada2/)
* [data.table](https://cran.r-project.org/web/packages/data.table/index.html)
* [dplyr](https://cran.r-project.org/web/packages/dplyr/index.html)
* [dunn.test](https://cran.r-project.org/web/packages/dunn.test/index.html)
* [ggpubr](https://cran.r-project.org/web/packages/ggpubr/index.html)
* [PairedData](https://cran.r-project.org/web/packages/PairedData/index.html)
* [pheatmap](https://cran.r-project.org/web/packages/pheatmap/)
* [plyr](https://cran.r-project.org/web/packages/plyr/index.html)
* [vioplot](https://cran.r-project.org/web/packages/vioplot/index.html)
* [rafalib](https://cran.r-project.org/web/packages/rafalib/index.html)
* [RColorBrewer](https://cran.r-project.org/web/packages/RColorBrewer/index.html)
* [tidyverse](https://www.tidyverse.org/packages/) (includes ggplot2, dplyr, tidyr, readr, purrr, tibble, stringr, etc.)
* [vegan](https://cran.r-project.org/web/packages/vegan/index.html)

I'm thinking we'll base our image on [`rocker/tidyverse`](https://hub.docker.com/r/rocker/tidyverse/). We should use a specific version when building to ensure reproducibility later on (currently latest available specific version is R `3.5.1`). That way we automatically get a web-based version of RStudio and a recent version of R and everything when we build our own image. I suggest we call our image `ctmrbio/rstudio` and tag it with whatever version of R was available when we built it, that way we can have newer versions in the future without too much trouble.

Let me know what you would like to have in our "standard CTMR R environment" @luhugerth @Steffi17 @StefPN @huyue87 @kimspindel !

**Comment by boulund** [comment]
I think our ambition should be to have a thorough base environment. However, I recognize that we of course might need some specialized packages for some analyses, but it should be possible to include a simple `install.packages(<packagename>)` in scripts that require that specialized package. 

If you identify a recurring need for some packages, we can build a new version of the our image that includes them.

**Comment by boulund** [comment]
Added Dockerfile for a standard CTMR R environment, called `ctmrbio/rstudio`. The `latest` tag points to the `ctmrbio/rstudio_R3.5.1_BioC3.7` image. There is an issue with the upstream images based on R `3.5.1`, where the `sm` package could not be installed (a dependency for `vioplot`), so I had to build this one using this slightly older version. 

The DADA2 version in this image is `1.11.0`, installed from Github sources today.
